image-patching/stopgap-script/nuage_overcloud_full_patch.py

```diff
@@ -392,9 +367,6 @@ def image_patching(args):
     if args.RhelUserName and args.RhelPassword and args.RhelPool:
         rhel_remove_subscription(args.ImageName)
 
-    cmds_run(['echo "Adding files post-patching"'])
-    add_files(args.ImageName, args.Version, workingDir)
-
     cmds_run(['echo "Done"'])
 
 
```

Comment:
We are not using Version any more, so remove it from line 421

Reply:
Will be fixed in next commit